mex-wholebodymodel/library/src/modelstate.cpp

```diff
@@ -20,62 +21,48 @@
 // global includes
 
 // library includes
-#include <mex.h>
-
 #include <iDynTree/ModelIO/URDFDofsImport.h>
-#include<yarpWholeBodyInterface/yarpWholeBodyModel.h>
+#include <yarpWholeBodyInterface/yarpWholeBodyModel.h>
 #include <yarpWholeBodyInterface/yarpWbiUtil.h>
-#include <yarp/os/Network.h>
 #include <yarp/os/ResourceFinder.h>
-#include <yarp/sig/Image.h>
 
 // local includes
 #include "modelstate.h"
 
 using namespace mexWBIComponent;
 
-ModelState* ModelState::modelState;
-wbi::iWholeBodyModel * ModelState::robotWBIModel = NULL;
+// ModelState *ModelState::modelState = NULL;
+boost::scoped_ptr<ModelState> ModelState::modelState(NULL);
```

Comment:
Introducing a boost dependency without declaring it in the CMake files is a wrong, as it could "misteriously" break the compilation in cases boost is not installed . 
Furthermore, requiring boost really complicates the installation process in Windows. Is this boost dependency really necessary?